tests/integration/test_definitions/spark/integration_tests.py

```diff
@@ -21,6 +21,13 @@
         user_flow_script="tests/integration/docusaurus/connecting_to_your_data/in_memory/spark_python_example.py",
         backend_dependencies=[BackendDependencies.SPARK],
     ),
+    IntegrationTestFixture(
+        name="how_to_configure_a_spark_datasource",
+        user_flow_script="tests/integration/docusaurus/connecting_to_your_data/datasource_configuration/how_to_configure_a_spark_datasource.py",
+        data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
+        data_dir="tests/test_sets/taxi_yellow_tripdata_samples/samples_2020",
+        backend_dependencies=[BackendDependencies.SPARK],
```

Comment:
For some reason this was never running with the `Spark` tests. Moved to appropriate place.